DOCS-11819-mqtt3-connector-final-kt (#1611)
mulesoft/docs-connectors: mqtt 3/1.0.0/modules/ROOT/pages/mqtt3-connector-examples.adoc

= MQTT 3 Connector Examples

To fully benefit from using Anypoint Connector for MQTT 3 (MQTT 3 Connector), see the following examples of configuring the connector:

* xref:mqtt3-connector-quality-of-service.adoc[Configure Quality of Service] +
Configure the quality of service (QoS) to specify levels of delivery assurance for inbound or outbound messages.
* xref:mqtt3-connector-lwt-message.adoc[Configure the Last Will And Testament Message] +

> Configure the last will and testament message to notify all the topic subscribers of the client’s disconnection from the broker.

Reviewer: ```suggestion
Configure the Last Will and Testament message to notify all the topic subscribers of the client’s disconnection from the broker.
```

* xref:mqtt3-connector-client-id.adoc[Set a client ID with MQTT 3 Connector] +

> Configure client IDs generators that help you set a client ID.

Reviewer: ```suggestion
Configure client ID generators that enable you to set a client ID.
```

* xref:mqtt3-connector-listener.adoc[Listen for New Messages] +

> Configure the *On New Message* source to listen for messages on a unique or multiple topics, and listen for messages on multiple topics using single-level or multi-level wildcards.

Reviewer: ```suggestion
Configure the *On New Message* source to listen for messages on a single or multiple topics, and listen for messages on multiple topics using single-level or multi-level wildcards.
```

* xref:mqtt3-connector-publish.adoc[Publish a Message] +

> Configure the *Publish* operation to publish a message to the desired topic and with a specific quality of service.

Reviewer: ```suggestion
Configure the *Publish* operation to publish a message to the desired topic with a specific quality of service.
```

* xref:mqtt3-connector-publish-receive.adoc[Listen and Publish Messages] +

> Configure the *On New Message* source and the *Publish* operation to listen and publish messages to listen and publish messages to a specific topic.

Reviewer: ```suggestion
Configure the *On New Message* source and the *Publish* operation to listen for and publish messages to a specific topic.
```
